ui-manchette/src/styles/operational-point.css

```
@@ -1,5 +1,8 @@
 .op {
   @apply text-grey-80;
+  height: 2rem;
+  line-height: 1.5rem;
+  padding-block: 0 0.3125rem;
```

To match the mockup vertical alignment

```suggestion
  padding-block: 3px 0.3125rem;
``

---

seems too high, but fine with 1.5px